packages/combobox/test/combobox.test.ts

```diff
@@ -917,6 +918,85 @@ describe('Combobox', () => {
         });
     });
 
+    describe('disabled items', () => {
+        it('disabled items should be disabled', async () => {
+            const el = await withDisabledItemsFixture();
+            await elementUpdated(el);
+
+            const opened = oneEvent(el, 'sp-opened');
+            el.click();
+            await opened;
+
+            const menuItems = el.shadowRoot.querySelectorAll('sp-menu-item');
+            expect(menuItems[2].disabled).to.be.true;
+        });
+        it('disabled items should not be focusable using keyboard', async () => {
+            const el = await withDisabledItemsFixture();
```

Comment:
dont initialize the el in all the test cases. You can globally define `el` and use it in the below test cases. Saves you lines of code.